packages/block-library/src/separator/style.scss

```diff
@@ -15,7 +15,7 @@
 		background: none !important;
 		border: none;
 		text-align: center;
-		width: none;
+		width: auto;
```

Comment:
I think this line could be removed given `width: none` has had no effect for a while now.

Reply:
Removed